src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -212,6 +212,11 @@ public static Mullion Wrap(Autodesk.Revit.DB.Mullion ele, bool isRevitOwned)
            return Mullion.FromExisting(ele, isRevitOwned);
         }
 
+        public static DetailCurve Wrap(Autodesk.Revit.DB.DetailCurve ele, bool isRevitOwned)
```

Comment:
I suggest to keep this as internal or private if it will be used, because I guess users will be confused by a node named "Wrap".